test/helm/chart-test.sh

```diff
@@ -314,20 +357,27 @@ get_chart_test_config() {
     echo "$config"
 }
 
+test_termination_nodes() {
+    if [[ $REUSE_ENV == false ]]; then
+        mkdir -p $TMP_DIR
+        install_kind
+        create_kind_cluster
+    fi
+
+    build_and_load_image
+    install_helm
```

Comment:
looks like you might need an `install_kubectl` here too

Reply:
:( yes